Assets/GameAssets/UI/UIScripts/InteractableUI.cs

```diff
@@ -0,0 +1,28 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class InteractableUI : MonoBehaviour
+{
+    private GameObject _interactUI;
```

Comment:
personally I would serialize this to get the exact interaction UI without having to worry if someone switches the name of the gameobject. I only really use GameObject.FindWithTag("blahBlah")